src/invidious/frontend/channel_page.cr

```diff
@@ -5,6 +5,8 @@ module Invidious::Frontend::ChannelPage
     Videos
     Shorts
     Streams
+    Podcasts
+    Releases
```

Comment:
Because you added new entries in this enum, you need to edit the `channel.ecr` file to handle them (even if you don't plan to implement the UI part).

Reply:
It should already be in the UI 
![image](https://github.com/iv-org/invidious/assets/78101139/dec3f7ef-32c0-4b41-bac7-d0d725b3cbcb)

Reply:
Oh, right, it's automatic! You still need to add the new pages here:
https://github.com/iv-org/invidious/blob/507bed6313b49564e53b69a5c9b4d072d1e05e4b/src/invidious/views/channel.ecr#L6-L14